packages/amplify-category-api/src/provider-utils/awscloudformation/cfn-api-artifact-handler.ts

```diff
@@ -59,6 +59,7 @@ class CfnApiArtifactHandler implements ApiArtifactHandler {
     const stacksDirectoryPath = path.join(resourceDir, stacksDirName);
     if (!fs.existsSync(stacksDirectoryPath)) {
       fs.mkdirSync(stacksDirectoryPath);
+      fs.copyFileSync(path.join(rootAssetDir, 'resolver-readme', 'RESOLVER_README.md'), path.join(resolverDirectoryPath, 'RESOLVER_README.md'));
```

Comment:
nit: we can still copy it to a file named `README.md`

Reply:
> 
> 
> nit: we can still copy it to a file named `README.md`

Hi, I'm sorry, could you clarify a bit regarding this? If we copy it to a file named `README.md`, then it will cause [e2e test failures](https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/4047/workflows/76311252-695b-4d19-84fb-0e02b23ed078) as per [this](https://github.com/aws-amplify/amplify-cli/pull/6845#issuecomment-794634835), right?

Reply:
No, only the file in the repo needs to be named RESOLVER_README.md. When we copy it to the customer's project, we can name the copied file whatever we want without affecting the tests

Reply:
Oh, got it. Thanks. I'll make the required changes 👍🏻